api/Parser.cpp

```diff
@@ -517,6 +517,32 @@ std::vector<ConferenceEndpoint> parseConferenceEndpoints(const nlohmann::json& r
     return endpoints;
 }
 
+ConferenceEndpointExtendedInfo parseEndpointExtendedInfo(const nlohmann::json& data)
+{
+    ConferenceEndpointExtendedInfo endpoint;
+    endpoint.basicEndpointInfo = parseConferenceEndpoint(data);
+    nlohmann::json iceSelectedTuple = nlohmann::json::object();
+
+    setIfExistsOrThrow<>(iceSelectedTuple, data, "iceSelectedTuple");
+    setIfExistsOrThrow<>(endpoint.localIP, iceSelectedTuple, "localIP");
+    setIfExistsOrThrow<>(endpoint.remoteIP, iceSelectedTuple, "remoteIP");
+    setIfExistsOrThrow<>(endpoint.localPort, iceSelectedTuple, "localPort");
+    setIfExistsOrThrow<>(endpoint.remotePort, iceSelectedTuple, "remotePort");
+    setIfExistsOrThrow<>(endpoint.protocol, iceSelectedTuple, "protocol");
+
+    for (const auto& it : requiredJsonArray(data, "audioUsidToSsrcMap"))
+    {
+        for (const auto& inner : it.items())
+        {
+            auto usid = inner.key();
```

Comment:
[minor] Avoid a `std:string` copy
```suggestion
            const auto& usid = inner.key();
```

Reply:
Fixed.